unittest/unittest.py

```diff
@@ -180,6 +180,12 @@ def __init__(self):
     def wasSuccessful(self):
         return self.errorsNum == 0 and self.failuresNum == 0
 
+
+def print_exc():
+    e = sys.exc_info()[1]
+    sys.print_exception(e, sys.stdout)
```

Comment:
Not all platforms have `print_exception` so that makes this a hard error on those.